### DISCUSSION TITLE
SLAMseq and whole transcriptome sequencing

### DISCUSSION THREAD

**Discussion by knight-lau** [nextflow-io/nextflow#4035] [Q&A]
Has anybody done SLAMseq analysis for whole transcriptome sequencing, i.e. not using quant seq as this is only for mature mRNA, but using an alternative sequencing method?

**Comment by mribeirodantas** [comment]
Hello, @knight-lau. This type of discussion is more appropriate in the Nextflow Slack. It's unlikely you'll get replies here. Try [this](https://nextflow.slack.com/archives/C03L6DMAN) instead, please.
